src/Workspaces/Core/Portable/SymbolId/SymbolKey.TupleTypeSymbolKey.cs

```diff
@@ -14,14 +14,14 @@ internal abstract partial class SymbolKey
     {
         private class TupleTypeSymbolKey : AbstractSymbolKey<TupleTypeSymbolKey>
         {
-            private readonly SymbolKey[] _types;
+            private readonly SymbolKey _underlyingType;
             private readonly string[] _names;
 
             internal TupleTypeSymbolKey(INamedTypeSymbol symbol, Visitor visitor)
             {
                 Debug.Assert(symbol.IsTupleType);
 
-                _types = symbol.TupleElementTypes.Select(t => GetOrCreate(t, visitor)).ToArray();
+                _underlyingType = GetOrCreate(symbol, visitor);
```

Comment:
I would expect refactoring tests to fail because of that. Do they?

---
In reply to: [62426511](https://github.com/dotnet/roslyn/pull/11160#discussion_r62426511) [](ancestors = 62426511)